docs/ref/contrib/admin/index.txt

```diff
@@ -1037,6 +1037,33 @@ subclass::
     If you need to specify a dynamic order (for example depending on user or
     language) you can implement a :meth:`~ModelAdmin.get_ordering` method.
 
+    .. note::
+
+        Admin changelist adds ``pk`` to ordering if it cannot find single field
+        or unique together set of fields providing a total ordering, to ensure
+        that a deterministic order is used across all database backends.
+        For example if you have a model like this::
+
+          from django.db import models
+
+          class Author(models.Model):
+              name = models.CharField(max_length=100)
+
+              class Meta:
+                  ordering = ['name']
+
+        The objects in admin changelist are sorted by ``name`` and ``pk``.
+        It could have serious performance issues is if you have:
+
+        * a lot of rows
+        * an index on the sort_col
```

Comment:
What does `sort_col` refer to?

Reply:
The column which we want to sort result based on it. I got it from [17198#comment:2](https://code.djangoproject.com/ticket/17198#comment:2).